zoo-css-challenge/style.css

```
@@ -55,9 +58,13 @@ h3 {
   font-size: 150%;
 }
 
+p {max-width: 700px;
+   margin: 25% ;}
```

```suggestion
   margin: auto;}
```
We can also use `auto` which centres the element. `auto` distributes the space left and right equally